packages/circuit-ui/components/Checkbox/Checkbox.tsx

```diff
@@ -37,6 +38,10 @@ export interface CheckboxProps extends HTMLProps<HTMLInputElement> {
    * Warning or error message, displayed in a tooltip.
    */
   validationHint?: string;
+  /**
+   * Removes the default bottom margin from the input.
```

Comment:
Yes, it's technically an input, but still I think it would make more sense to call it checkbox here

```suggestion
   * Removes the default bottom margin from the checkbox.
```